xml/Microsoft.VisualBasic.ApplicationServices/AuthenticationMode.xml

```diff
@@ -17,7 +17,7 @@
 ## Remarks  
  You can set the authentication mode for an application by passing one of this enumeration's values to the <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.%23ctor%2A> constructor. For more information, see [Extending the Visual Basic Application Model](~/docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md).  
   
- A Visual Basic application that uses the Visual Basic Application Model can specify whether it uses Windows authentication or custom authentication. You can also configure the authentication mode in the Visual Studio Integrated Development Environment. For more information, see [Application Page, Project Designer (Visual Basic)](../Topic/Application%20Page,%20Project%20Designer%20\(Visual%20Basic\).md).  
+ A Visual Basic application that uses the Visual Basic Application Model can specify whether it uses Windows authentication or custom authentication. You can also configure the authentication mode in the Visual Studio Integrated Development Environment. For more information, see [Application Page, Project Designer (Visual Basic)](http://msdn.microsoft.com/library/8cec9fea-cd92-47ff-88dd-7c928f0b4a74).  
```

Comment:
The topic exists in docs: https://docs.microsoft.com/en-us/visualstudio/ide/reference/application-page-project-designer-visual-basic